### PR TITLE
scatter capacity vs curtailment

### PR DESCRIPTION
**Pull request by BainanXia**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Integrate another plot function from spot check notebook: scatter plot capacity vs curtailment

### What the code is doing
- Implement plot function `plot_scatter_capacity_vs_curtailment`.
- Add demo notebook for the new plot function.

### Testing
No tests is added. The plot function calls existing functions in `analyze` module for calculations.

### Where to look
```
/postreise/plot/plot_scatter_capacity_vs_curtailment.py
/postreise/plot/demo/scatter_capacity_vs_curtailment_demo.ipynb
```

### Usage Example/Visuals
Demos can be found in the demo notebook.

### Time estimate
10 min. It is similar with `plot_scatter_capacity_vs_capacity_factor`.

### CODE REVIEW

File: postreise/plot/plot_scatter_capacity_vs_curtailment.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    dayofweek=None,
+    markersize=50,
+    fontsize=20,
+    title=None,
+    percentage=False,
+    show_plot=True,
+):
+    """Generate for a given scenario the scatter plot of the capacity (x-axis) vs
+        curtailment as a fraction of available resources (y-axis) of generators
+        located in area and fueled by resources over a time range.
```
Comment (rouille) [inline postreise/plot/plot_scatter_capacity_vs_curtailment.py:28]: The last two lines should not be indented:
```
"""Generate for a given scenario the scatter plot of the capacity (x-axis) vs
curtailment as a fraction of available resources (y-axis) of generators located 
in area and fueled by resources over a time range.
```

Comment (BainanXia) [reply postreise/plot/plot_scatter_capacity_vs_curtailment.py]: Done.

**Comment by rouille** [comment]
All the other notebooks file name in the **demo** finish with `_demo`. Do we want to rename ` postreise/plot/demo/scatter_capacity_vs_curtailment.ipynb` for consistency?

**Comment by BainanXia** [comment]
> All the other notebooks file name in the **demo** finish with `_demo`. Do we want to rename ` postreise/plot/demo/scatter_capacity_vs_curtailment.ipynb` for consistency?

Done.

**Review by rouille — APPROVED** [review]
The notebook runs smoothly and the code looks good

**Pull request merged**